truss/templates/server/model_wrapper.py

```diff
@@ -149,29 +152,51 @@ def try_load(self):
             bundled_packages_path = Path("/packages")
             if bundled_packages_path.exists():
                 sys.path.append(str(bundled_packages_path))
-        model_module_name = str(
-            Path(self._config["model_class_filename"]).with_suffix("")
-        )
 
-        module = importlib.import_module(
-            f"{self._config['model_module_dir']}.{model_module_name}"
-        )
-        model_class = getattr(module, self._config["model_class_name"])
-        model_class_signature = inspect.signature(model_class)
-        model_init_params = {}
-        if _signature_accepts_keyword_arg(model_class_signature, "config"):
-            model_init_params["config"] = self._config
-        if _signature_accepts_keyword_arg(model_class_signature, "data_dir"):
-            model_init_params["data_dir"] = data_dir
-        if _signature_accepts_keyword_arg(model_class_signature, "secrets"):
-            model_init_params["secrets"] = SecretsResolver.get_secrets(self._config)
-        if _signature_accepts_keyword_arg(model_class_signature, "lazy_data_resolver"):
-            model_init_params["lazy_data_resolver"] = LazyDataResolver(data_dir).fetch()
+        secrets_resolver = SecretsResolver.get_secrets(self._config)
+        lazy_data_resolver = LazyDataResolver(data_dir).fetch()
```

Comment:
fetch() is what actually performs the resolution (and returns None). So the difference between this and the existing code is that this will always try to resolve bptrs, regardless of if the model class signature accepts it - which should be fine, since LazyDataResolver can handle there being no bptr manifest.

Looking at the code I also noticed we also have always been assigning model_init_params["lazy_data_resolver"] = None (when model class signature accepts the resolver). I assume we just use it for a check like `"lazy_data_resolver" in {"lazy_data_resolver": None}` then.